src/components/RecommendedFile.vue

```diff
@@ -98,6 +114,18 @@ export default {
 		isFolder() {
 			return this.mimeType === 'httpd/unix-directory'
 		},
+		description() {
+			if (this.reason === 'recently-edited') {
+				return t('recommendations', 'Last updated {timeAgo}', { timeAgo: this.formattedTime })
+			}
+			if (this.reason === 'recently-shared') {
+				return t('recommendations', 'Shared with you {timeAgo}', { timeAgo: this.formattedTime })
+			}
+			if (this.reason === 'recently-commented') {
+				return t('recommendations', 'Last commented on {timeAgo}', { timeAgo: this.formattedTime })
+			}
+			return null
```

Comment:
So timestamp must be available as a prop as otherwise this will fail if timestamp is null but reason is one of those.